File: cpp/inicio/TiposDados.cpp
// Lib de entrada e saida de dados;
#include <iostream>

// Lib para formatar numero de casas decimais
#include <iomanip>

// Lib para usar funcoes de manipulacao de texto
#include <string>

// Lib universal para nao ter que ficar add cada uma das libs
// #include <bits/stdc++.h>

using namespace std;

int main () {

    int idade;
    double salario, altura;
    char genero;
    string nome;

    idade = 20;
    salario = 5800.0;
    altura = 1.63;
    genero = 'F';
    nome = "Maria Silva";

    // cmd para fixar numero de casas decimais
    cout << fixed << setprecision(2);

    cout << "IDADE = " << idade << endl;
    cout << "SALARIO = " << salario << endl;
    cout << "ALTURA = " << altura << endl;
    cout << "GENERO = " << genero << endl;
    cout << "NOME = " << nome << endl;

    return 0;

}

File: java/curso-logica-java-nelio/TiposDados.java
import java.util.Locale;

public class Main {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		
		int idade;
		double salario, altura;
		char genero;
		String nome;
		
		idade = 20;
		salario = 5800.0;
		altura = 1.63;
		genero = 'F';
		nome = "Maria Silva";
		
		System.out.println("IDADE = " + idade);
		
		// 2 casas decimais
		System.out.println("SALARIO = " + String.format("%.2f", salario));
		System.out.println("ALTURA = " + String.format("%.2f", altura));
		
		System.out.println("GENERO = " + genero);
		System.out.println("NOME = " + nome);
	}
}